### PR TITLE
fix: Fix calendar event included in the workflow, doesn't show the meeting link

### LINKED ISSUE #13820: [CAL-3199] [CAL-3191] Calendar event included in the workflow, doesn't show the meeting link
The link shows in the workflow description however in the calendar event it says integrations:google:meet 

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3191](https://linear.app/calcom/issue/CAL-3191/calendar-event-included-in-the-workflow-doesnt-show-the-meeting-link)</sub>

<sub>[CAL-3199](https://linear.app/calcom/issue/CAL-3199/[cal-3191]-calendar-event-included-in-the-workflow-doesnt-show-the)</sub>

### PR DESCRIPTION
**Pull request by asadath1395**
## What does this PR do?

Fixes #13820 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Install the google calendar app and google meet
2. Create a new event type
3. Create a new workflow which triggers when booking happens or some actions takes place
4. Check the workflow generated email for the attachment and check the location

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by CarinaWolli** [comment]
@asadath1395  Can you fix the type error (https://github.com/calcom/cal.com/actions/runs/8052096213/job/21991559784?pr=13872)
I am coming back to review this soon 🙏

**Comment by asadath1395** [comment]
Hey @CarinaWolli,

I have fixed the type error, for some reason my VS code wasn't showing the error so i thought all is well 😃

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -2291,8 +2291,7 @@ async function handler(
     loggerWithEventDetails.error("Error while creating booking references", JSON.stringify({ error }));
   }
 
-  const metadataFromEvent = videoCallUrl ? { videoCallUrl } : undefined;
```
Comment (CarinaWolli) [inline packages/features/bookings/lib/handleNewBooking.ts:2294]: fixed this existing bug, it didn't use the public video call url

**Review by CarinaWolli — APPROVED** [review]
Made some small changes, looks good now 🙌🏻

Thank you for your contribution 🙏

**Pull request merged**